<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HVNLT052" ignored="0" name="FirstTest" tests="1" failures="1" timestamp="21 Nov 2018 07:05:46 GMT" time="8.15" errors="0">
  <testcase name="login" time="0.012" classname="tests.LoginTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat java.util.Objects.requireNonNull(Objects.java:203)at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:106)at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:85)at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)at pages.BasePage.<init>(BasePage.java:23)at pages.HomePage.<init>(HomePage.java:12)at tests.LoginTests.login(LoginTests.java:24)at com.sun.proxy.$Proxy1.stop(Unknown Source)at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)at java.lang.Thread.run(Thread.java:748)... Removed 46 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- FirstTest -->
